client/rest-high-level/src/test/java/org/elasticsearch/client/ClusterClientIT.java

```diff
@@ -178,15 +175,19 @@ public void testClusterHealthYellowClusterLevel() throws IOException {
 
         logger.info("Shard stats\n{}", EntityUtils.toString(
                 client().performRequest(new Request("GET", "/_cat/shards")).getEntity()));
-        assertYellowShards(response);
```

Comment:
removed this line because it's duplicated from another test that covers this case, see method below.